admin_guide/manage_rbac.adoc

```diff
@@ -403,7 +403,7 @@ Subjects:
   Group	system:unauthenticated
 
 
-Name:		self-provisioners
+Name:		self-provisioner
```

Comment:
@stuartchuan, based on the command output that you posted, this change is valid:
```
# oc get clusterrole | grep self-provisioner
self-provisioner
````